### PR TITLE
Change email addresses for tickets for niedersachcsen

### PR DESCRIPTION
**Pull request by robertstrz**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-6727

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: theme/n21/views/help/mail-card.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p>Manche Anliegen brauchen mehr Möglichkeiten, als die Felder der Kontaktfunktion zulassen. Schreib uns eine E-Mail, wenn du mit uns ins Gespräch kommen willst:</p>
 <div id="email-addresses">
-    <a href="mailto:feedback@hpi-schul-cloud.de" target="_blank" rel="noopener">feedback@hpi-schul-cloud.de</a> ans HPI für Ideen, Kritik und Wünsche rund um die Cloud selbst. <br>
+    <a href="mailto:nbc-wunsch@netz-21.de" target="_blank" rel="noopener">nbc-wunsch@netz-21.de</a> ans HPI für Ideen, Kritik und Wünsche rund um die Cloud selbst. <br>
```
Comment (ABialoskorski) [inline theme/n21/views/help/mail-card.hbs:3]: shouldn't it be nbc-support@netz-21.de ?

File: theme/n21/views/lib/extended_footer.hbs [hunk 2]
```diff
@@ -4,8 +4,8 @@
 			<div  class="col-md-4">
 				<h4>{{$t "global.link.contact" }}</h4>
 				<ul>
-					<li><a href="mailto:feedback@hpi-schul-cloud.de" target="_blank" rel="noopener" aria-label="Support-Anfragen {{$t "global.link.openInNewTabWarning"}}">Support-Anfragen:  <span class="text-nowrap">feedback@hpi-schul-cloud.de</span></a></li>
-					<li><a href="mailto:nbc-support@netz-21.de" target="_blank" rel="noopener" aria-label="Allgemeine Anfragen {{$t "global.link.openInNewTabWarning"}}">Allgemeine Anfragen: <span class="text-nowrap">nbc-support@netz-21.de</span></a></li>
+					<li><a href="mailto:nbc-wunsch@netz-21.de" target="_blank" rel="noopener" aria-label="Support-Anfragen {{$t "global.link.openInNewTabWarning"}}">Feedback/Wünsche:  <span class="text-nowrap">nbc-wunsch@netz-21.de</span></a></li>
```
Comment (ABialoskorski) [inline theme/n21/views/lib/extended_footer.hbs:7]: shouldn't it be nbc-support@netz-21.de ?

**Comment by ABialoskorski** [comment]
please add changelog and update version number ;)

**Review by ABialoskorski — APPROVED** [review]

File: theme/n21/views/lib/extended_footer.hbs (hunk 2 above)
Comment (robertstrz) [reply theme/n21/views/lib/extended_footer.hbs:7]: This email is below

**Review by LukasSielski — APPROVED** [review]

File: theme/n21/views/help/mail-card.hbs (hunk 1 above)
Comment (robertstrz) [reply theme/n21/views/help/mail-card.hbs:3]: this one is below ;) they should be different

**Comment by robertstrz** [comment]
Things, that was changed only in NBC:
image


image

**Comment by luizzaly** [comment]
I am confused, I did not see the E-mail-address nbc-wunsch@netz-21.de mentioned in the ticket...?

**Comment by robertstrz** [comment]
I have discussed adding this email address "nbc-wunsch@netz-21.de" with @schurigh

**Review by luizzaly — APPROVED** [review]

**Review by monaweitzenberg — APPROVED** [review]

**Pull request merged**